### DISCUSSION TITLE
How can I call Routes.Content.Add with an internal folder in the program

### DISCUSSION THREAD

**Discussion by CrhistopherDiaz** [dotnet/WatsonWebserver#112] [Q&A] [answered]
How can I call Routes.Content.Add with an internal folder in the program

**Comment by jchristn** [comment]
You have to have access to the ```Server``` instance within the method from which you wish to call ```Server.Routes.Content.Add```.

**Reply by CrhistopherDiaz** [reply]
could you give me an example please

**Reply by CrhistopherDiaz** [reply]
I say something like this ResourceDictionary Source="/Web/Home.xaml"/ that calls that file which is internal

**Reply by jchristn** [reply]
I'm not completely understanding what you are asking.

You can't call ```Routes.Content.Add``` from a file - it has to be from within a method in your program that has access to the instance of the ```Server``` object.

If you're asking about adding a folder as a content route, you can use ```Routes.Content.Add("./folder/", true);```.  That will make files inside of ```./folder/``` available.

**Reply by jchristn** [reply]
BTW, ```./folder/``` will look for a directory called ```folder``` in the directory in which your app was executed.

**Reply by CrhistopherDiaz** [reply]
> BTW, will look for a directory called in the directory in which your app was executed.`./folder/``folder`

Let me see if what I want to do is call a folder that I create in the same project, that is, I create a folder in the project and add files, what I say is call that folder so that it is shown which is inside the program

**Comment by jchristn** [comment, marked as answer]
Yes, just right-click each file in the project that you want to be included as output and say "Copy to Output"
